core/frontend/src/App.vue

```diff
@@ -41,97 +36,116 @@
       </v-app-bar>
     </v-card>
 
-    <v-navigation-drawer
-      v-model="drawer"
-      absolute
-      temporary
+    <v-row
+      class="fill-height"
+      no-gutters
     >
-      <v-container
-        elevation="0"
-        class="d-flex justify-center"
+      <v-navigation-drawer
+        id="navigator-drawer"
+        permanent
+        :expand-on-hover="is_small_screen"
```

Comment:
```suggestion
        :expand-on-hover="$vuetify.breakpoint.mdAndDown"
```

Reply:
we shold use breakpoints where we can:
https://vuetifyjs.com/en/features/breakpoints/#thresholds

Reply:
Good catch